include/ipmitool/ipmi_sdr.h

```diff
@@ -30,8 +30,8 @@
  * EVEN IF SUN HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
  */
 
-#ifndef IPMI_SDR_H
-#define IPMI_SDR_H
+#ifndef IPMITOOL_IPMI_SDR_H
+#define IPMITOOL_IPMI_SDR_H
```

Comment:
This is a great finding, but if Wikipedia is to be trusted, `#pragma once` is nowadays supported by all compilers on all systems, so it would be better to switch to it as long as we're changing every include file anyway.

See https://en.wikipedia.org/wiki/Pragma_once#Portability

Reply:
Nice suggestion. Addressed.